.github/workflows/release.yml

```diff
@@ -184,9 +184,9 @@ jobs:
     - name: Generate sources
       run: sudo bash flatpak/gen-sources.sh
     - name: Flatpak
-      run: sudo flatpak-builder --repo=repo build-dir flatpak/uk.co.mrbenshef.TeX-Match.json
```

Comment:
You could go one step further and make the github job use the flatpak github action, I can do that in a separate PR otherwise :)

Reply:
I barely speak GitLab CI,  your the flathub/GitHubAction maestro :-)

Of course you could make a PR to my fork to roll it all together

Reply:
I originally used the flatpak github action, but I found it quite a bit slower than doing it manually. Docker overhead I guess? Then again the windows build will always be 30+ minuets on release anyway so that probably doesn't matter.

Reply:
Hmm, I don't see why it should be slower. Given that you have to download the FreeDesktop & Rust SDK's along with all the dependencies compared to just pulling a docker image that already contains the Freedesktop SDK (20.08) installed on it.  

I will give it a try and see :)

Reply:
Resolved in #74